README.md

```diff
@@ -66,10 +66,8 @@ git clone https://github.com/ethereum/remix-project.git
 * Build `remix-project`:
 ```bash
 cd remix-project
-yarn install
-yarn run build:libs // Build remix libs
-nx build
-nx serve
+yarn
+yarn run build
 ```
 
 Open `http://127.0.0.1:8080` in your browser to load Remix IDE locally.
```

Comment:
`yarn serve` will be required to run the server

Reply:
Also should we mention when you are doing mostly frontend work, you can do yarn serve:hot to have hmr enabled so you don't have too many refreshes?

Reply:
ok, updated